examples/pytorch/IterativePruning/README.md

```diff
@@ -0,0 +1,77 @@
+## Iterative Pruning
+Pruning is the process of compressing a neural network that involves removing weights from a trained model.
+It could be achieved by either removing the neurons in a specific layer or make the weights of the connections 
+that are nearer to zero as zero. The script is an example of later. Pruning a model has an impact on
+accuracy of the model, while it makes the model lightwieght. In this example, we train a Alexnet model to classify 
+CIFAR 10 dataset. Basemodel along with the parameters, metrics and summary are stored in mlflow.
+Subsequently, the trained Alexnet model stored in mlflow is downloaded and pruned iteratively by using the custom 
+inputs provided from the cli. AXClient is used to provide the initial pruning percentage as well as decides the number
```

Comment:
@ankan94 Please add more description about Ax and link to it and describe it Ax will be used for the adaptive experiments for pruning

Reply:
Also mention that objective function that is being used for the Adaptive experimentation, so that people understand the benefit of using Ax.